diff_drive_controller/include/diff_drive_controller/diff_drive_controller.h

```diff
@@ -229,10 +229,12 @@ namespace diff_drive_controller{
 
     realtime_tools::RealtimeBuffer<DynamicParams> dynamic_params_;
 
-    /// Dynamic Reconfigure server
+
     typedef dynamic_reconfigure::Server<DiffDriveControllerConfig> ReconfigureServer;
-    
-    std::shared_ptr<ReconfigureServer> dyn_reconf_server_;
+    /// To avoid Dynamic Reconfigure Server warning
+    boost::recursive_mutex mDynServerMutex;
```

Comment:
Please always follow the reigning coding style when contributing to a codebase.
```suggestion
    boost::recursive_mutex dyn_server_mutex_;
```

Reply:
Or better yet, to match the name of server, `dyn_reconf_server_mutex_`